src/index.ts

```diff
@@ -131,6 +133,15 @@ const sign = (privateKey: string, message: string): Signature => {
 };
 
 const verify = (publicKey: string, message: string, signature: Signature) => {
+  if (useCryptoCpp) {
+    return starkVerifyCpp(
+      BigInt(publicKey),
+      BigInt(`0x${message}`),
```

Comment:
Same :warning: Is it fine to assume that hexa message strings will never start with `0x`? At least it's the case in tests

Reply:
Same.